build.gradle

```diff
@@ -9,12 +9,15 @@ buildscript {
 		maven { url 'http://repo.springsource.org/plugins-release' }
 	}
 	dependencies {
-		classpath 'org.springframework.build.gradle:docbook-reference-plugin:0.2.8'
+		classpath 'io.spring.gradle:docbook-reference-plugin:0.3.0'
+		classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.0'
```

Comment:
Aren't we going to go ahead with the newer `asciidoctor` plugin version: https://github.com/rwinch/asciidoctor-gradle-examples/blob/master/asciidoc-to-all-example/build.gradle?